contrib/packs/actions/pack_mgmt/setup_virtualenv.py

```diff
@@ -85,7 +85,8 @@ def run(self, packs, update=False, python3=False):
 
         for pack_name in packs:
             setup_pack_virtualenv(pack_name=pack_name, update=update, logger=self.logger,
-                                  proxy_config=self.proxy_config, use_python3=python3)
+                                  proxy_config=self.proxy_config, use_python3=python3,
+                                  no_download=True)
```

Comment:
Don't you mean for this to use the `kwarg`?

Reply:
Not sure I understand what you mean - I'm indeed passing this new argument to the function as a keyword argument :)